<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelecommandWidget</class>
 <widget class="QWidget" name="TelecommandWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Telecommand</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="addressLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="autoScrollCheckBox">
       <property name="text">
        <string>Auto Scroll</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="reconnectCheckBox">
       <property name="text">
        <string>Reconnect on Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="clearCheckBox">
       <property name="text">
        <string>Clear on Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clearPushButton">
       <property name="text">
        <string>Clear Log</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="statusPushButton">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="standbyPushButton">
       <property name="text">
        <string>Standby</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetPushButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="dateTImeLabel">
       <property name="text">
        <string>Date &amp; Time (UTC):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy HH:mm:ss</string>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="latitudeLabel">
       <property name="text">
        <string>Latitude:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="latitudeDoubleSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>8</number>
       </property>
       <property name="minimum">
        <double>-90.000000000000000</double>
       </property>
       <property name="maximum">
        <double>90.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000001000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="longitudeLabel">
       <property name="text">
        <string>Longitude:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="longitudeDoubleSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>8</number>
       </property>
       <property name="minimum">
        <double>-180.000000000000000</double>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000001000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="heightLabel">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="heightDoubleSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>8</number>
       </property>
       <property name="maximum">
        <double>500000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000001000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="hotStartPushButton">
       <property name="text">
        <string>Hot Start</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="coldStartPushButton">
       <property name="text">
        <string>Cold Start</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="warmStartPushButton">
       <property name="text">
        <string>Warm Start</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="connectPushButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
